src/artemis/devices/system_tests/test_oav_system.py

```diff
@@ -31,3 +31,11 @@ def test_grid_overlay():
     snapshot_directory = "."
     RE = RunEngine()
     RE(take_snapshot_with_grid(oav, snapshot_filename, snapshot_directory))
+
+
+@pytest.mark.s03
+def test_get_zoom_levels():
+    my_zoom_controller = ZoomController("BL03I-EA-OAV-01:FZOOM:", name="test_zoom")
+    my_zoom_controller.wait_for_connection()
+    # raise Exception(my_zoom_controller.allowed_zoom_levels)
```

Comment:
Should: Can you remove commented out code please?

Reply:
Done, forgot to delete it before committing.